crates/tauri-cli/Cargo.toml

```diff
@@ -149,3 +150,5 @@ native-tls = [
 ]
 native-tls-vendored = ["native-tls", "tauri-bundler/native-tls-vendored"]
 rustls = ["tauri-bundler/rustls", "cargo-mobile2/rustls", "ureq/rustls"]
+config-json5 = ["json5"]
+config-toml = []
```

Comment:
no need to add these features. This is the CLI and not shipped with the final user app so we can always include them.

Reply:
oh ok thanks